docs/markdoc/index.ts

```diff
@@ -33,16 +33,26 @@ class MarkdocValidationFailure extends Error {
   }
 }
 
-export type DocContent = {
+export type DocsContent = {
   content: Tag;
   title: string;
   description: string;
 };
 
-export async function readDocContent(filepath: string): Promise<DocContent> {
+export type BlogContent = BlogFrontmatter & {
+  content: Tag;
+};
+
+export async function readBlogContent(filepath: string): Promise<BlogContent> {
+  let content = await fs.readFile(filepath, 'utf8');
+  const frontmatter = extractBlogFrontmatter(content);
+  return { content: transformContent(`blog/${filepath}`, content), ...frontmatter };
```

Comment:
```suggestion
  return { content: transformContent(`docs/${filepath}`, content), ...frontmatter };
```
(docs refers to the top-level directory)